pegasus/helpers/form_helpers.rb

```diff
@@ -183,8 +187,10 @@ def update_form(kind, secret, data)
   symbolized_data = Hash[data.map{|k, v| [k.to_sym, v]}]
   data = validate_form(kind, prev_data.merge(symbolized_data))
 
+  normalized_email = data[:email_s].to_s.strip.downcase if data.key?(:email_s)
```

Comment:
Do these two normalizations need to keep in sync? Might want to pull out a `normalize_email` helper if so